hearpreprocess/pipeline.py

```diff
@@ -44,6 +44,46 @@
 }
 
 
+def _diagnose_split_labels(taskname, event_str, df, filename_col):
+    """Makes split and label diagnostics"""
+    assert "split" in df.columns
+    assert "label" in df.columns
+    assert filename_col in df.columns
+    diagnostics.info(
+        f"{taskname} File count in each split {event_str}: "
+        "{}".format(df.groupby("split")[filename_col].nunique().to_dict())
+    )
+    # Get the unique labels in each split
+    split_label = df.groupby("split")["label"].nunique().to_dict()
+    # Get percentage of rows with a particular label for each split
+    split_label_percentage = {
+        split: (
+            round(
+                split_df["label"].value_counts(normalize=True) * 100.0,
+                2,
+            )
+        ).to_dict()
```

Comment:
Use a dictionary comprehension. These long things that black breaks up are hard to read so please figure out a way to clean them up